Profiling: add upgrade notice of known issues for 8.9 versions (#3214)
elastic/observability-docs: docs/en/observability/profiling-get-started.asciidoc

IMPORTANT: When upgrading, you must remove all existing profiling data.
We still recommend upgrading as the latest version contains several improvements and new features.

[discrete]
[[profiling-upgrade-known-issues]]

> ==== Known issue during the configuration of data ingestion

Reviewer: ```suggestion
==== Known issue when configuring data ingestion

```

> If you created your {ecloud} deployment starting from a 7.x version or earlier, and you are currently running it on 8.9.0, 8.9.1 or 8.9.2, you may encounter a problem while trying to enable Universal Profiling.

Reviewer: ```suggestion
If your {ecloud} deployment originated from version 7.x or earlier and is currently running on version 8.9.0–8.9.2, you may encounter a problem while trying to enable Universal Profiling.

```

> 

Reviewer: ```suggestion
```

> Specifically, clicking the 'Set up Universal Profiling' button triggers an error message that reads **Error while setting up Universal Profiling**.

Reviewer: ```suggestion
Specifically, clicking the *Set up Universal Profiling* button triggers an error message that reads `Error while setting up Universal Profiling`.

```

> To rectify this situation, an upgrade to version 8.9.3 or above is needed.

Reviewer: ```suggestion
Upgrade to version 8.9.3 or more recent to fix this issue.

```
